linux_os/guide/system/permissions/files/dir_system_commands_root_owned/oval/shared.xml

```diff
@@ -0,0 +1,45 @@
+<def-group>
+  <definition class="compliance" id="dir_system_commands_root_owned" version="1">
+    {{{ oval_metadata("
+        Checks that directories /bin /sbin /usr/bin /usr/sbin /usr/local/bin /usr/local/sbin
+        are owned by root.
+      ") }}}
+    <criteria >
+      <criterion test_ref="test_ownership_system_commands_dirs" />
+    </criteria>
+  </definition>
+
+  <unix:file_test  check="all" check_existence="none_exist" comment="system commands directories uid root" id="test_ownership_system_commands_dirs" version="1">
+    <unix:object object_ref="object_system_commands_dirs_ownership" />
+  </unix:file_test>
+
+  <!-- Access to /var/log/audit rule regex-->
+  <constant_variable id="commands_directories_regex" version="1" datatype="string" comment="command directories list">
+       <value>^\/[s?]bin|^\/usr\/[s?]bin|^\/usr\/local\/[s?]bin</value>
+  </constant_variable>
+
+  
+  <unix:file_object comment="system commands directories have root ownership" id="object_system_commands_dirs_ownership" version="1">
+    <!-- Check that bin /sbin /usr/bin /usr/sbin /usr/local/bin /usr/local/sbin directories belong to user with uid 0 (root) -->
+    <unix:behaviors recurse="directories" recurse_direction="down" max_depth="-1" recurse_file_system="all" />
```

Comment:
I don't think yozu want recurse_depth="-1" here. This would recurse all directories within /usr/bin etc. Is that what you are trying to accomplish?
```suggestion
    <unix:behaviors recurse="directories" recurse_direction="down" max_depth="0" recurse_file_system="all" />
```